web-api/migration-terraform/main/lambdas/migration-segments.js

```diff
@@ -51,16 +54,21 @@ const migrateRecords = async ({
     items = await bugMigration0036(items);
   }
 
-  if (!ranMigrations['0036-phone-number-format.js']) {
+  if (!ranMigrations['migration0036.js']) {
     applicationContext.logger.debug('about to run migration 0036');
-    items = await migration0036(items);
+    items = await migration0036(items, documentClient);
```

Comment:
Looks like this is also an unintentional change — I don’t believe this migration takes a second argument.